app/src/main/java/com/duckduckgo/app/autocomplete/api/AutoComplete.kt

```diff
@@ -68,18 +83,4 @@ open class AutoCompleteApi @Inject constructor(
             .toList()
             .onErrorReturn { emptyList() }
             .toObservable()
-
-    data class AutoCompleteResult(
-        val query: String,
-        val suggestions: List<AutoCompleteSuggestion>,
-        val hasBookmarks: Boolean
-    )
-
-    sealed class AutoCompleteSuggestion(val phrase: String, val suggestionType: Int) {
-        class AutoCompleteSearchSuggestion(phrase: String, val isUrl: Boolean) :
-            AutoCompleteSuggestion(phrase, SUGGESTION_TYPE)
-
-        class AutoCompleteBookmarkSuggestion(phrase: String, val title: String, val url: String) :
-            AutoCompleteSuggestion(phrase, BOOKMARK_TYPE)
```

Comment:
This was moved, however worth noting that the adapter display type info (e.g `BOOKMARK_TYPE`) was removed from class definitions and is only now only used by the adapter.